cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ResponseUtils.java

```diff
@@ -45,7 +45,7 @@ static void setResponseCode(HttpServletResponse response, int code, boolean json
     if (corsEnabled) {
```

Comment:
Is this the only method which sends out headers for all types of responses CC emits ?

Reply:
There are 3 config parameters CC supports in terms of CORS

```
webserver.http.cors.enabled=false
webserver.http.cors.origin=http://localhost:8080/
webserver.http.cors.allowmethods=OPTIONS,GET,POST
webserver.http.cors.exposeheaders=User-Task-ID
```

The behavior of these parameters control how the following responses are sent

1. HTTP `OPTIONS` Request (made automatically by browser)

This should send the following headers in response based on `webserver.http.cors.enabled`

```
Access-Control-Allow-Origin: $(webserver.http.cors.origin)
Access-Control-Allow-Methods: $(webserver.http.cors.allowmethods)
Access-Control-Allow-Headers: $(webserver.http.cors.exposeheaders)
Access-Control-Max-Age: 1728000
```

2. HTTP `GET/POST` Requests

CC should send the following headers in response based on `webserver.http.cors.enabled`

```
Access-Control-Allow-Origin: $(webserver.http.cors.origin)
Access-Control-Expose-Headers: $(webserver.http.cors.exposeheaders)
```

References: 
https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers

Reply:
Yes, this methods is shared by all the response logic, i.e normal response, error response, in-progress response.